### PR TITLE
#1141 implemented Negligence class

### PR DESCRIPTION
**Pull request by krzyk**
#1141 
* implemented delay and add methods on negligence class, added tests for them
* updated datum version (which contains the negligence.xsd)
* fixed people.xml (it contained old datum version)
* todo for continuation

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1424](https://github.com/zerocracy/farm/issues/1424) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1424](https://github.com/zerocracy/farm/issues/1424) is assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1941556067); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by krzyk** [comment]
@paulodamaso ping

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by krzyk** [comment]
@paulodamaso could you review this?

File: src/main/java/com/zerocracy/pmo/Negligence.java [hunk 1]
```diff
@@ -18,8 +18,11 @@
...
 import com.zerocracy.Farm;
 import com.zerocracy.Item;
+import com.zerocracy.Project;
 import com.zerocracy.Xocument;
 import java.io.IOException;
+import java.time.Instant;
+import org.xembly.Directives;
 
 /**
  * Negligence. This metric calculates how many times a user has lost
```
Comment (paulodamaso) [inline src/main/java/com/zerocracy/pmo/Negligence.java]: @krzyk I think you should update this `@todo` title and description: it should reference #1141 and have its test updated according what have been done in #1141.

File: src/main/java/com/zerocracy/pmo/Negligence.java [hunk 2]
```diff
@@ -69,10 +72,37 @@ public Negligence(final Pmo pkt, final String user) {
...
+
+    /**
+     * Add a negligence.
+     * @param proj Project
+     * @param job Job id
+     * @throws IOException If fails
      */
-    public int delays() {
-        throw new UnsupportedOperationException("Not yet implemented!");
+    public void add(final Project proj, final String job) throws IOException {
```
Comment (paulodamaso) [inline src/main/java/com/zerocracy/pmo/Negligence.java]: @krzyk Do we really need to pass `Project` and `Job` here? I mean, is there a way that some job belongs to more than one project? What does it happen if we pass a `Project` here and some job that doesn't belongs to this `Project`?

File: src/test/resources/com/zerocracy/bundles/refresh_person_job_count_on_agenda_update/people.xml [hunk 3]
```diff
@@ -15,7 +15,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 -->
-<people xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://datum.zerocracy.com/0.61.6/xsd/pmo/people.xsd" version="1" updated="2016-12-29T09:03:21.684Z">
+<people xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://datum.zerocracy.com/0.62.2/xsd/pmo/people.xsd" version="1" updated="2016-12-29T09:03:21.684Z">
```
Comment (paulodamaso) [inline src/test/resources/com/zerocracy/bundles/refresh_person_job_count_on_agenda_update/people.xml]: @krzyk This date shouldn't be updated with the actual date of schema update?

Comment (krzyk) [reply src/test/resources/com/zerocracy/bundles/refresh_person_job_count_on_agenda_update/people.xml]: @paulodamaso this doesn't matter for the tests (and I don't know if it matters at all :)

File: src/main/java/com/zerocracy/pmo/Negligence.java (hunk 2 above)
Comment (krzyk) [reply src/main/java/com/zerocracy/pmo/Negligence.java]: @paulodamaso It is how all other similar metrics are designed (Blanks, Speed, Verbosity), it was designed in datum so here we need to use those values. You could report a bug there.

File: src/main/java/com/zerocracy/pmo/Negligence.java (hunk 1 above)
Comment (krzyk) [reply src/main/java/com/zerocracy/pmo/Negligence.java]: @paulodamaso good catch :)

**Comment by krzyk** [comment]
@paulodamaso I've added the todo, please take a look

File: src/test/resources/com/zerocracy/bundles/refresh_person_job_count_on_agenda_update/people.xml (hunk 3 above)
Comment (krzyk) [reply src/test/resources/com/zerocracy/bundles/refresh_person_job_count_on_agenda_update/people.xml]: @paulodamaso I assume this is to mark when the xml was updated, not when schema was updated.
With schema update to people I had to add there something, in tests it doesn't matter what date is there.

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1424](https://github.com/zerocracy/farm/issues/1424) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +31 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**